app/assets/stylesheets/application.scss

```diff
@@ -16,6 +16,30 @@
 
  */
 
- .tattoo_image {
-   width: 150px;
- }
+body {
+  background: url(/background.jpg) no-repeat center center fixed;
+  -webkit-background-size: cover;
```

Comment:
Properties should be ordered background, -moz-background-size, -o-background-size, -webkit-background-size, background-size